locale/en/about/index.md

```diff
@@ -40,7 +40,7 @@ If some of this language is unfamiliar, there is a full article on
 ---
 
 Node.js is similar in design to, and influenced by, systems like Ruby's
-[Event Machine][] or Python's [Twisted][]. Node.js takes the event model a bit
+[Event Machine][] and Python's [Twisted][]. Node.js takes the event model a bit
```

Comment:
I think both conjunctions are idiomatic here. I don't really see much benefit to changing it, unless it makes it more understandable for people for whom English is not their primary language. I won't block this if other's thing it's a good change.

Honestly, I suspect it would be best to rewrite this paragraph to eliminate the references to Ruby and Python. Those probably made more sense when those technologies were vastly more well-known than Node.js. Right now, the references to Ruby and Python don't clarify anything. It's just extra stuff. I very much doubt that there are many readers who see EventMachine or Twisted and then say, "Oh, I see, I understand how those technologies are designed and now Node.js makes a lot more sense to me."

Perhaps an indication of how not-particularly-illuminating the references are, there's a typo: Ruby's is `EventMachine` and not `Event Machine`, but no one has apparently thought to say something until right now.